### ISSUE TITLE
Please add tag/key propery to EasyDialog

### ISSUE THREAD

**Issue by danvinokour** [michaelye/EasyDialog#21]
This is going to be used to identify the tooltip if i want to know which one is displaying or dismissed and so on...

**Comment by michaelye** [comment]
@danvinokour Yes,I think I should add Tag for every version.Good advice.Thanks~

**Comment by danvinokour** [comment]
Hi,
just to make sure..i meant to add someting like `String key;` to the EasyDialog class so when i use it and i have 10 EasyDialogs i would know who is who. like view has `setTag();`

**Comment by michaelye** [comment]
@danvinokour Ok,I misunderstand before,I got it now.Currently, EasyDialog only support one tooltip every time,and can not support more than one EasyDialog View at the same time.I think more than one tooltip is not often used, so I need to consider about it,Thx~
